sdk-api-src/content/handleapi/nf-handleapi-duplicatehandle.md

```diff
@@ -76,30 +76,32 @@ The handle to be duplicated. This is an open object handle that is valid in the
 
 A handle to the process that is to receive the duplicated handle. The handle must have the PROCESS_DUP_HANDLE access right.
 
+This parameter is optional and can be specified as NULL if the **DUPLICATE_CLOSE_SOURCE** flag is set in _Options_.
```

Comment:
My reasons to assume this is correct:
* The code samples I mentioned in the description, among others, support the claim that the parameter can be set to **NULL** to avoid actual handle duplication.
* The last section in the remarks that guides how "To close a duplicated handle from the source process" guides to set `lpTargetHandle` to **NULL**, but doesn't mention `hTargetProcessHandle`. From all of the other details I deduced that `hTargetProcessHandle` was meant to be set to **NULL** as well, otherwise it doesn't line up with the documentation of the `lpTargetHandle` parameter which recommends not to set it to **NULL.**
* `DuplicateHandle` passes its parameters almost as-is to `ZwDuplicateObject`, which documents this behavior. In fact, this line is copied verbatim from [that documentation](https://docs.microsoft.com/en-us/windows-hardware/drivers/ddi/ntifs/nf-ntifs-zwduplicateobject).